xml/System.Runtime.InteropServices.Marshalling/PointerArrayMarshaller`1.xml

```diff
@@ -148,9 +166,17 @@
         <Parameter Name="value" Type="System.Byte*" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="value">The native value.</param>
+        <summary>Sets the native value representing the array.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+<xref:System.Runtime.InteropServices.Marshalling.CustomTypeMarshallerFeatures.TwoStageMarshalling>
```

Comment:
PHantom xref here and elsewhere in this file.

Reply:
Fixed https://github.com/dotnet/dotnet-api-docs/pull/8185/files#r914241547